### DISCUSSION TITLE
'run-p' is not recognized as an internal or external command, operable program or batch file.- Windows 10

### DISCUSSION THREAD

**Discussion by MBennettLowe** [codebuddies/frontend#149] [Q&A] [answered]
Yesterday while installing the dependencies for the Front End portion of the project command "npm i npm-run-all"  I received the error,  "run-p' is not recognized as an internal or external command,operable program or batch file" . I retrieved installing this dependency today and it appeared to have installed without error.

**Comment by lpatmo** [comment, marked as answer]
See: https://github.com/codebuddies/frontend/discussions/149

**Reply by bhuvanaswami** [reply]
Where can I get the answer for this?

**Reply by Wilhop** [reply]
I think @Ipatmo meant https://github.com/codebuddies/frontend/discussions/148
